apps/ashley/templates/services/nginx/dc.yml.j2

```
@@ -32,6 +32,10 @@ spec:
                   values:
                   - "ashley-nginx-{{ deployment_stamp }}"
               topologyKey: kubernetes.io/hostname
+{% if ashley_nginx_image_pull_secret_name is defined and ashley_nginx_image_pull_secret_name is not none %}
+      imagePullSecrets:
+        - name: "{{ ashley_nginx_image_pull_secret_name }}"
```

Shouldn't we fallback to a global secret (e.g. organization's dockerhub account)?

```suggestion
        - name: "{{ ashley_nginx_image_pull_secret_name | default(default_image_pull_secret_name) }}"
```

---

good idea but the `if` has to be modified as well :sweat_smile:

---

@jmaupetit I refactored to allow a `default_image_pull_secret_name` variable as you suggested :pray: